### ISSUE TITLE
missing key length in Mysql.php

### ISSUE THREAD

**Issue by Busfreak** [BlueTeck/kanboard_plugin_metadata#11]
The new version throws an error while migrating DB:
column 'machine_name' used in key specification without a key length

The key length has to be defined with TEXT columns

`UNIQUE(machine_name(255), attached_to(255)`

or something similar.

Configuration

```
Version: master
PHP version: 5.3.10-1ubuntu3.21
PHP SAPI: apache2handler
OS version: Linux 3.2.0-97-generic-pae
Datenbanktreiber mysql
Database version: 5.5.46-0ubuntu0.12.04.2
Browser: Mozilla/5.0 (Windows NT 6.1; WOW64; rv:44.0) Gecko/20100101 Firefox/44.0
```

**Comment by fatrex** [comment]
It's a typo in the schema file, due to copy/paste :P I will fix this in the next commit :)

**Comment by fatrex** [comment]
This should be fixed in 71b96e7
@Busfreak can you try again?

**Comment by crabnky** [comment]
I had this error as well and last commit fixed it. Thanks!
